src/Components/Navigation/SidePhoto.js

```diff
@@ -1,17 +1,11 @@
-import { Show, Image, Box } from "@chakra-ui/react";
+import { Show, Image, Box, useColorModeValue } from "@chakra-ui/react";
 
 const SidePhoto = () => {
+  const photo = useColorModeValue("https://i.ibb.co/Q6JNRKg/light-mode.jpg", "https://i.ibb.co/sQ5S8zy/dark-mode.jpg")
   return (
     <Show above="xl">
-      <Box
-        mt="-16"
-        right={0}
-        zIndex={-2}
-        pos={"fixed"}
-        borderLeft={"1px"}
-        borderColor={"gray.700"}
-      >
-        <Image src="https://i.ibb.co/FqbjLw6/dark-banner-2.png"></Image>
+      <Box mt="-16" right={0} zIndex={-2} w="20%" pos={"fixed"} X>
```

Comment:
```suggestion
      <Box mt="-16" right={0} zIndex={-2} w="20%" pos={"fixed"}  
      h="100%" 
      bgSize="100%" 
      bgImage={`url(${photo})`}
      backgroundRepeat="repeat"
      bgPosition="top">
      X // is this a typo?
```

this should make the image fill the right 20% of the page from top to bottom at all sizes.

Reply:
Nice catch, and great job on making it a background photo!